projects/plugins/jetpack/modules/search/class-jetpack-instant-search.php

```diff
@@ -12,6 +12,41 @@
  * @since 8.3.0
  */
 class Jetpack_Instant_Search extends Jetpack_Search {
+	/**
+	 * The name of instant search sidebar
+	 *
+	 * @since 9.8.0
+	 *
+	 * @var string
+	 */
+	const JETPACK_INSTANT_SEARCH_SIDEBAR = 'jetpack-instant-search-sidebar';
+
+	/**
+	 * Variable to save old sidebars_widgets value
```

Comment:
Could we describe when this variable is saved/deleted? Describing the action sequence for `after_switch_theme` and `pre_update_option_sidebars_widgets` could be useful for future contributors.

Reply:
```
The value is set when action `after_switch_theme` is applied and cleared on filter `pre_update_option_sidebars_widgets`.
The filters mentioned above run on /wp-admin/themes.php?activated=true, a request closely following switching theme.
```

Added comments in 4fcef02